spec/spec_helper.rb

```diff
@@ -25,4 +25,5 @@
 VCR.configure do |c|
   c.cassette_library_dir = "fixtures/vcr_cassettes"
   c.hook_into :webmock
+  c.allow_http_connections_when_no_cassette = true
```

Comment:
To be consistent with our other tests and to make sure our tests aren't reliant on an external service, we should create a cassette for this rather enabling "no cassette" mode.